pkg/httphelper/client.go

```diff
@@ -585,6 +586,33 @@ func (client *NodeMgmtClient) CallCompactionEndpoint(pod *corev1.Pod, compactReq
 	return nil
 }
 
+// CallTSReloadEndpoint calls the async version of TSReload
+func (client *NodeMgmtClient) CallinodeTsReloadEndpoint(pod *corev1.Pod) error {
+	client.Log.Info(
+		"calling Management API TS REload endpoint - POST /api/v0/node/encryption/internode/truststore/reload",
+		"pod", pod.Name,
+	)
+	podHost, podPort, err := BuildPodHostFromPod(pod)
+	if err != nil {
+		return err
+	}
+
+	request := nodeMgmtRequest{
+		endpoint: "/api/v0/ops/node/encryption/internode/truststore/reload",
+		host:     podHost,
+		port:     podPort,
+		method:   http.MethodPost,
+		timeout:  60 * time.Second,
```

Comment:
question: Do you happen to know how long this operation usually takes? If it's closer to 60 seconds, we probably should have an async endpoint done also in the future instead of relying cass-operator to do the async operation.

Reply:
I have no idea, no.